src/input/source/hidraw/dualsense.rs

```diff
@@ -469,7 +469,6 @@ pub const CAPABILITIES: &[Capability] = &[
     Capability::Gamepad(Gamepad::Button(GamepadButton::LeftStickTouch)),
     Capability::Gamepad(Gamepad::Button(GamepadButton::LeftTrigger)),
     Capability::Gamepad(Gamepad::Button(GamepadButton::North)),
-    Capability::Gamepad(Gamepad::Button(GamepadButton::QuickAccess)),
```

Comment:
This looks like it was supposed to be
`    Capability::Gamepad(Gamepad::Button(GamepadButton::Mute)),` instead

Reply:
oh interesting, I missed that